core/iwasm/aot/aot_runtime.c

```diff
@@ -3151,9 +3151,11 @@ aot_alloc_frame(WASMExecEnv *exec_env, uint32 func_index)
             }
         }
 
-        if (func_type->param_count == 0) {
+        if (func_type->param_cell_num < 2) {
             /* We reserve at least two cells for native frame */
-            frame_ref[0] = frame_ref[1] = 0;
+            for (; j < 2; j++) {
+                frame_ref[j] = 0;
+            }
```

Comment:
In 32-bit, when param_cell_num = 1 and the first param is GC ref, frame_ref[0] should be 1? How about:
```C
    for (j = func_type->param_cell_num; j < 2; j++) {
         frame_ref[j] = 0;
    }
```

Reply:
Done, thanks!